draft-ietf-quic-transport.md

```diff
@@ -1228,45 +1228,49 @@ version as part of the transport parameters (see {{transport-parameters}}).  As
 a result, modification of version negotiation packets by an attacker can be
 detected.
 
-The client includes two fields in the transport parameters:
-
-* The negotiated_version is the version that was finally selected for use.  This
-  MUST be identical to the value that is on the packet that carries the
-  ClientHello.  A server that receives a negotiated_version that does not match
-  the version of QUIC that is in use MUST terminate the connection with a
-  VERSION_NEGOTIATION_ERROR error code.
-
-* The initial_version is the version that the client initially attempted to use.
-  If the server did not send a version negotiation packet {{packet-version}},
-  this will be identical to the negotiated_version.
+The client includes the initial_version field in its transport parameters.  The
+initial_version is the version that the client initially attempted to use.  If
+the server did not send a version negotiation packet {{packet-version}}, this
+will be identical to the negotiated_version.
```

Comment:
To the negotiated_version in the encrypted_extensions.

Reply:
I'm not allowed to use TLS message names, but I'll use "server transport parameters".